<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsEscalationEditorDlgClass</class>
 <widget class="QDialog" name="CwmsEscalationEditorDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eskalationsschritt bearbeiten</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Eskalationsschritt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="m_pqsbEscalationStep">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ãœberschreitung der max. Abarbeitungsdauer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="m_pqsbLimit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Zu informierende Mitarbeiter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="m_pqtwEmployees">
        <column>
         <property name="text">
          <string>Vorname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nachname</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="CwmsUserSelectionBox" name="m_pUsers"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_pqpbAdd">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../WMSadmin/resources/emresources.qrc">
            <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pqpbRemove">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../WMSadmin/resources/emresources.qrc">
            <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOk">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Eskalationsschritt bearbeiten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CwmsUserSelectionBox</class>
   <extends>QComboBox</extends>
   <header location="global">CwmsUserSelectionBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsEscalationEditorDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAdd</sender>
   <signal>clicked()</signal>
   <receiver>CwmsEscalationEditorDlgClass</receiver>
   <slot>AddEmployeeClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemove</sender>
   <signal>clicked()</signal>
   <receiver>CwmsEscalationEditorDlgClass</receiver>
   <slot>RemoveEmployeeClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOk</sender>
   <signal>clicked()</signal>
   <receiver>CwmsEscalationEditorDlgClass</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddEmployeeClickedSlot()</slot>
  <slot>RemoveEmployeeClickedSlot()</slot>
  <slot>OKClickedSlot()</slot>
 </slots>
</ui>
